dynamodump/dynamodump.py

```diff
@@ -1117,6 +1135,13 @@ def main():
         "backups (defaults to use '" + str(DATA_DUMP) + "') [optional]",
         default=str(DATA_DUMP),
     )
+    parser.add_argument(
+        "--billingMode",
+        help="Set billing mode between PROVISIONED|PAY_PER_REQUEST "
```

Comment:
Same here. And also probably create a new variable for `PAY_PER_REQUEST`

Reply:
Do you mean it's better to have `--provisioned` and `--payByRequest` instead of `--billingMode` option ?

Reply:
Add something like `PAY_PER_REQUEST_BILLING_MODE="PAY_PER_REQUEST"` and all the `PROVISIONED` and `PAY_PER_REQUEST` arbitrary strings in the parser help and choices can reference those

Reply:
Ok, got it and fixed